docs/index.md

```diff
@@ -368,6 +380,21 @@ This cookie takes precedence over the `JWT_AUTH_COOKIE`. If you have both cookie
 
 Impersonation cookies use the `JWT_AUTH_COOKIE_*` settings.
 
+### JWT_DELETE_STALE_BLACKLISTED_TOKENS
+
+Enables deleting of stale blacklisted tokens on `post_save` when set to `True`. The date & time
+ before which the tokens should be deleted is defined in
+  `JWT_STALE_BLACKLISTED_TOKEN_EXPIRATION_TIME`.
+
+Default is `False`.
+
+### JWT_STALE_BLACKLISTED_TOKEN_EXPIRATION_TIME
+
+This is an instance of Python's `datetime.datetime`. Represents the date & time before which expired
+ blacklisted tokens should be deleted.
+
+Default is `timezone.now()`(timezone aware current moment).
```

Comment:
Is it always aware or does it depend on the `USE_TZ` setting?

Reply:
The value depends on the `USE_TZ` setting. Will be repaired in the next commit.